app/assets/stylesheets/taxa/form.css

```diff
@@ -36,3 +36,35 @@ fieldset.column {margin-bottom: 1em;}
   width: 100%;
   height: 3em;
 }
+
+.jqmWindow {
+  display: none;
+  position: fixed;
+  top: 10%;
+  left: 50%;
+  margin-left: -350px;
+  width: 700px;
+  background-color: #EEE;
+  color: #333;
+  border: 1px solid black;
+  padding: 20px;
+}
+
+.jqmOverlay { background-color: #000; }
+
+* html .jqmWindow {
+  position: absolute;
+  top: expression( ( document.documentElement.scrollTop || document.body.scrollTop ) + Math.round( 17 * ( document.documentElement.offsetHeight || document.body.clientHeight ) / 100 ) + 'px' );
```

Comment:
Don't use expression() in CSS. This is an Internet Explorer-specific feature that apparently some other browsers picked up and then IE abandoned. Just write Javascript in the .js files.

Reply:
I just removed it - are we developing for IE anymore?

Reply:
I think it's worth supporting Edge, but not IE. It's certainly not worth using functionality that was originally made to only work in IE.